package.json

```diff
@@ -25,7 +25,8 @@
     "build:common": "cross-env NODE_ENV=production webpack --config config/webpack.config.common.js --progress --hide-modules",
     "prepublish": "npm run build",
     "commit": "cz",
-    "enableGitHooks": "simple-git-hooks"
+    "enableGitHooks": "simple-git-hooks",
+    "publish": "clean-publish"
```

Comment:
Please rename it to `cleanPublish`.

`yarn publish` will call default publish command. With `cleanPublish` we can run it as `yarn cleanPublish`